metadata_test.go

```diff
@@ -149,14 +138,22 @@ func TestLoadMetadata(t *testing.T) {
 			s := sandbox.New(t)
 			s.BuildTree(tcase.layout)
 
-			metadata, err := terramate.LoadMetadata(s.RootDir())
-
+			stackEntries, err := terramate.ListStacks(s.RootDir())
 			if tcase.wantErr != nil {
 				assert.IsError(t, err, tcase.wantErr)
 			}
 
-			if diff := cmp.Diff(tcase.want, metadata, cmpopts.IgnoreUnexported(tcase.want)); diff != "" {
-				t.Fatalf("want %+v != got %+v.\ndiff:\n%s", tcase.want, metadata, diff)
+			gotMetadata := []stack.Metadata{}
```

Comment:
I kept this `metadata_test.go` just to make sure metadata is loaded in the same way as before but this better to be moved to stack tests in the future.

Reply:
I was wondering about this, thanks for the comment :100: